src/phoenix/server/app.py

```diff
@@ -218,9 +221,20 @@ async def check_healthz(_: Request) -> PlainTextResponse:
     return PlainTextResponse("OK")
 
 
+@router.get("/openapi.json")
+async def openapi() -> Dict[str, Any]:
+    return get_openapi_schema()
+
+
 @router.get("/docs")
 async def api_docs() -> Response:
-    return get_swagger_ui_html(openapi_url="/openapi.json", title="arize-phoenix API")
+    return get_swagger_ui_html(
+        openapi_url="/openapi.json",  # this json file is served as a static asset
+        title="arize-phoenix API",
+        swagger_ui_parameters={
+            "defaultModelsExpandDepth": -1,  # hides the schema section in the Swagger UI
+        },
```

Comment:
this is part of fastapi natively right? I guess there's a bit of customization. But we probably could remove?

Reply:
Do you mean remove the copied implementation of `get_swagger_ui_html`? I undid the change we made for hiding private tags, so at this point, our implementation is the same as the fastapi implementation. I left since it seems like there's the desire to potentially customize the Swagger UI, but I can remove.

Reply:
Removed in c3c04bdd48fff6bf5cb891b9e909e4a5fa5dd9ea